standard/abstract_tests/ATS_class_core.adoc

```diff
@@ -3,7 +3,7 @@
 [%metadata]
 label:: http://www.wmo.int/spec/wcmp/2.0/conf/core
 subject:: Requirements Class "core"
-classification:: Target Type:Discovery Metadata
+classification:: Target Type: Discovery Metadata
```

Comment:
This needs to stay as is, as per OGC ModSpec templating (https://raw.githubusercontent.com/opengeospatial/templates/master/standard/template/abstract_tests/ATS_class_example1.adoc)